Website/HTML/pulse_poem.html

```diff
@@ -1,236 +1,183 @@
+                  <p>The idea of this poem was to communicate that "silence is deafening," the absence
+                     of
+                     a heartbeat growing louder and lourder. A person close to the narrator of the poem
+                     is dying, and their growing feeling of grief in represented through the symbol of
+                     sound.</p>
+                  
                   <p>I used an increasing tempo between stanzas to create a sense of escalation and
                      stress, and attempted to simulate a heartbeat by using repetition of the word
                      itself.</p>
-               </div>
-               <!-- <div id="themes">Themes:
+                  </div>
+               <div id="themes">Themes: 
                   <ul>
-
+                     
                      <li>Death/Loss</li>
-
+                     
                      <li>Grief</li>
-
+                     
                      <li>Sound</li>
-                  </ul>
-               </div> -->
+                     </ul>
+               </div>
             </div>
          </section>
-         <div id="poem_holder">
-
+         <section id="poem_holder">
             <div id="poem">
-               <div id="stanzaNum1" class="stanza">
-
-                  <div id="lineNum1" class="line heartbeat_line">Heartbeat.</div>
-
-                  <div id="lineNum2" class="line">A thunderously loud</div>
-
-                  <div id="lineNum3" class="line">rhythmic thumping to </div>
-
-                  <div id="lineNum4" class="line">the soul-giving pumping, </div>
-
-                  <div id="lineNum5" class="line">bloody beat of</div>
-
-                  <div id="lineNum6" class="line">life itself.</div>
-               </div>
-
-
-               <div id="stanzaNum2" class="stanza">
-
-                  <div id="lineNum7" class="line heartbeat_line">Heartbeat.</div>
-
-                  <div id="lineNum8" class="line">Shockingly dim and</div>
-
-                  <div id="lineNum9" class="line">a whimpering in the</div>
-
-                  <div id="lineGroupNum1" class="line_group"
-                     onclick="toggleLineInfo('line_info_point1')" style="cursor: pointer">
-
-                     <div id="lineNum10" class="line">
-                        <span class="lineGroupNum1">empty cavern of a</span>
-                     </div>
-
-                     <div id="lineNum11" class="line">
-                        <span class="lineGroupNum1">cavity left cold and</span>
-                     </div>
-
-                     <div id="lineNum12" class="line">
-                        <span class="lineGroupNum1">decaying.</span>
-                     </div>
+               
+               <div id="stanzaNums-1" class="stanza">
+                  
+                  <div id="lineNum-l-1" class="line">Heartbeat.</div>
+                  
+                  <div id="lineNum-l-2" class="line">A thunderously loud</div>
```

Comment:
I set a hyphen in between "lineNum" and the `@xml:id` you're pulling so it should be easy to split on the first hyphen. Only, I'm not sure we want two hyphens? Possibly we'll want to make the `@xml:id` not use a hyphen... (just `l2`, etc`) for ease of splitting and matching pieces of these values in your JavaScript!